src/main/java/robot/subsystems/drivetrain/pure_pursuit/Point.java

```diff
@@ -80,4 +80,24 @@ public double getY() {
     public void setY(double y) {
         this.y = y;
     }
+
+    @Override
+    public String toString() {
+        return
+                x +
```

Comment:
A little nitpicky but this seems a little weirdly formatted

Reply:
I agree.

Reply:
oh yeah i see what you are saying, fixed it now